perf_test/sparse/KokkosSparse_spmv.cpp

```diff
@@ -79,258 +79,145 @@
 #include <spmv/ArmPL_SPMV.hpp>
 #endif
 
-enum {KOKKOS, MKL, ARMPL, CUSPARSE, KK_KERNELS, KK_KERNELS_INSP, KK_INSP, OMP_STATIC, OMP_DYNAMIC, OMP_INSP};
```

Comment:
I see ARMPL explicitly added in the new file, but not the others (MKL, CUSPARSE, and the various other things, is there a reason?

Reply:
@srajama1 -- Again, we made these changes to accommodate Luc's new ARM - handling in this test.